compiler/rustc_mir_dataflow/src/elaborate_drops.rs

```diff
@@ -274,8 +274,7 @@ where
                 let tcx = self.tcx();
 
                 assert_eq!(self.elaborator.param_env().reveal(), Reveal::All);
-                let field_ty =
-                    tcx.normalize_erasing_regions(self.elaborator.param_env(), f.ty(tcx, substs));
```

Comment:
Do you know why the normalize was here in the first place?

Reply:
I don't know. The call to normalize dates back to 2016.